app/default/application_routes.py

```diff
@@ -0,0 +1,232 @@
+from datetime import datetime
+from functools import wraps
+from http.client import METHOD_NOT_ALLOWED
+import requests
+from app.constants import ApplicationStatus
+from app.default.data import get_account
+from app.default.data import get_application_data
+from app.default.data import get_fund_data
+from app.default.data import get_round_data
+from app.default.data import get_round_data_fail_gracefully
+from app.helpers import format_rehydrate_payload
+from app.helpers import get_token_to_return_to_application
+from app.models.application_summary import ApplicationSummary
+from config import Config
+from flask import abort
+from flask import Blueprint
+from flask import current_app
+from flask import g
+from flask import redirect
+from flask import render_template
+from flask import request
+from flask import url_for
+from flask_wtf import FlaskForm
+from fsd_utils.authentication.decorators import login_required
+from flask_babel import force_locale
+from app.default.routes import current_datetime_after_given, current_datetime_before_given
+
+application_bp = Blueprint("application_routes", __name__, template_folder="templates")
+
+def verify_application_owner_local(f):
```

Comment:
should this sit outside of routes?

Reply:
Yes there's a TODO I missed when copying for it to move to utils with a couple of other methods. Will do those in a separate PR.